articles/cognitive-services/text-analytics/quickstarts/python.md

```diff
@@ -1,420 +1,239 @@
+        text="I had a terrible time at the hotel. The staff was rude and the food was awful."),
+    models.MultiLanguageInput(id="3", language="es", 
+        text="Los caminos que llevan hasta Monte Rainier son espectaculares y hermosos."),
+    models.MultiLanguageInput(id="4", language="es", 
+        text="La carretera estaba atascada. Había mucho tráfico el día de ayer."),
+]
+
+sentiment_results = client.sentiment(documents=sentiment_docs)
 ```
 
-The sentiment API can now be used to analyze the documents for their sentiments.
+After the `sentiment` call, `sentiment_results.documents` is a list of `SentimentBatchResultItem` instances, each corresponding to a submitted document. The `SentimentBatchResultItem` includes a `score` attribute, which is the detected sentiment value. The Python code below displays the sentiment results as an HTML table.
 
 ```python
-headers   = {"Ocp-Apim-Subscription-Key": subscription_key}
-response  = requests.post(sentiment_api_url, headers=headers, json=documents)
-sentiments = response.json()
-pprint(sentiments)
-```
+from IPython.display import HTML  # or HTML = print if you aren't using a Jupyter notebook
+
+table = []
+header = "<tr><th>{}</th><th>{}</th><th>{}</th></tr>".format("ID", "Text", "Score")
 
-Successful JSON response:
+for doc, res in zip(sentiment_docs, sentiment_results.documents):
+    row = "<tr><td>{doc.id}</td><td>{doc.text}</td><td>{score:.3}</td></tr>".format(doc=doc, score=res.score)
+    table.append(row)
 
-```json
-{'documents': [{'id': '1', 'score': 0.7673527002334595},
-                {'id': '2', 'score': 0.18574094772338867},
-                {'id': '3', 'score': 0.5}],
-    'errors': []}
+HTML("<table>{0}{1}</table>".format(header, "\n".join(table)))
 ```
 
-The sentiment score for a document is between 0.0 and 1.0, with a higher score indicating a more positive sentiment.
-
-<a name="KeyPhraseExtraction"></a>
 
 ## Extract key phrases
 
-The Key Phrase Extraction API extracts key-phrases from a text document, using the [Key Phrases method](https://westus.dev.cognitive.microsoft.com/docs/services/TextAnalytics.V2.0/operations/56f30ceeeda5650db055a3c6). This section of the walkthrough extracts key phrases for both English and Spanish documents.
+The [`key_phrases` method](https://westus.dev.cognitive.microsoft.com/docs/services/TextAnalytics.V2.0/operations/56f30ceeeda5650db055a3c6) extracts key phrases from a text document.
 
-The service endpoint for the key-phrase extraction service is accessed via the following URL:
+A key phrase extraction call works much like a sentiment analysis call. Multiple "document can be submitted in a single call, and each document must have a unique ID within the set of documents in a given `key_phrases` call. 
 
-```python
-key_phrase_api_url = text_analytics_base_url + "keyPhrases"
-print(key_phrase_api_url)
-```
-    https://westcentralus.api.cognitive.microsoft.com/text/analytics/v2.0/keyPhrases
```

Comment:
Same here, all links needs to be updated to v2.1 of the API.

Reply:
See previous comment.